### DISCUSSION TITLE
Rotation gesture in normal map view

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#623] [Q&A] [answered]
I have tested Cruiser 5 for hiking and it works very well. The rendering speed is good and the Elevate theme looks convincing with the new map engine. 👍👍
But one feature from v4 I miss very much: Map rotation in normal map view with two-finger gesture. Map zoom exists, but rotation not... 
And the short tap on the compass to redirect the map towards North.
And the long tap on the compass for the submenu map orientation: Manual/by gps/by compass. Maybe you can add the option "Always North" that disables the rotation.
Please put it on your implemetation list. It was a good and "normal" map feature that is gone with v5, unfortunately...

**Comment by devemux86** [comment]
Map rotation works in navigation and follow location modes.
Long press on compass works when these modes are active.

I also plan to see the rotation gesture in the future.

---

Elevate and all these themes are designed in this map engine.
So they certainly work better in Cruiser 5.

**Reply by kds112** [reply]
Thanks for your reply. The point is that map rotation with two-finger gesture should also work in the normal map view, not only in follow and navigation mode... Hope it will come back in the future 😊

**Comment by devemux86** [comment, marked as answer]
- Rotation gesture is available in https://github.com/devemux86/cruiser/discussions/641
